runtime/jcl/common/reflecthelp.c

```diff
@@ -191,7 +185,7 @@ getFieldTypeAnnotationsAsByteArray(JNIEnv *env, jobject jlrField)
         J9JNIFieldID *fieldID = vmThread->javaVM->reflectFunctions.idFromFieldObject(vmThread, NULL, fieldObject);
     	U_32 *annotationData = getFieldTypeAnnotationsDataFromROMField(fieldID->field);
     	if ( NULL != annotationData ) {
-    		j9object_t annotationsByteArray = getAnnotationDataAsByteArray(vmThread, annotationData);
+			j9object_t annotationsByteArray = getAnnotationDataAsByteArray(vmThread, annotationData, NULL);
```

Comment:
Minor: tabbing looks off here

Reply:
This was changed because of a line ending check failure https://openj9-jenkins.osuosl.org/job/PullRequest-LineEndingsCheck-OpenJ9/11431/console

Reply:
I see, you can fix the whole function so it doesnt look off

Reply:
Fixed.